test/vanilla/__snapshots__/ignite-remove-demo.test.ts.snap

```diff
@@ -3,40 +3,66 @@
 exports[`ignite-cli remove-demo should print the expected response 1`] = `
 "   
    Removing demo code from '/user/home/ignite'
-   Found '@demo remove-file' in /user/home/ignite/.maestro/FavoritePodcast.yaml
-   Found '@demo remove-file' in /user/home/ignite/.maestro/Login.yaml
+removing file /user/home/ignite/.maestro/FavoritePodcast.yaml
```

Comment:
I think we should have a test like this for the addition of the --mst=false flag as well just to ensure that continues working